lib/moab/storage_object_validator.rb

```diff
@@ -203,12 +205,18 @@ def basename(path)
       path.split(File::SEPARATOR)[-1]
     end
 
-    def result_hash(response_code, addl = nil)
-      { response_code => error_code_msg(response_code, addl) }
+    def result_hash(response_code, msg_args = nil)
+      { response_code => error_code_msg(response_code, msg_args) }
     end
 
     def error_code_msg(response_code, addl = nil)
```

Comment:
Can you document the parameters here?  It seems like response code is a format string and addl is either a Hash or something, but it's unclear what that case is.